common/persistence/cassandraPersistence_test.go

```diff
@@ -690,13 +690,16 @@ func (s *cassandraPersistenceSuite) TestWorkflowMutableState_Activities() {
 		{
 			ScheduleID:               1,
 			ScheduledEvent:           []byte("scheduled_event_1"),
+			ScheduledTime:            currentTime,
 			StartedID:                2,
 			StartedEvent:             []byte("started_event_1"),
+			StartedTime:              currentTime,
 			ScheduleToCloseTimeout:   1,
 			ScheduleToStartTimeout:   2,
 			StartToCloseTimeout:      3,
 			HeartbeatTimeout:         4,
 			LastHeartBeatUpdatedTime: currentTime,
+			TimerTaskStatus:          10,
```

Comment:
The enum i defined in history engine layer instead of here, thinking very specific to timer builder logic. hence this test didn't have access to that enum, so it is using a +ve value.